pkg/etekcity_device.py

```diff
@@ -44,19 +44,90 @@ def __init__(self, adapter, _id, vesync_dev):
             },
             self.on)
 
-        self.properties['power'] = EtekcityProperty(
+        t = threading.Thread(target=self.poll)
+        t.daemon = True
+        t.start()
+
+    def poll(self):
+        """Poll the device for changes."""
+        while True:
+            time.sleep(_POLL_INTERVAL)
+            self.vesync_dev.update()
+
+            for prop in self.properties.values():
+                prop.update()
+
+    @property
+    def on(self):
+        """Determine whether or not the device is on."""
+        return self.vesync_dev.device_status == 'on'
+
+
+class EtekcityBulb(EtekcityDevice):
+    """Etekcity bulb type."""
+
+    def __init__(self, adapter, _id, vesync_dev):
+        """
+        Initialize the object.
+
+        adapter -- the Adapter managing this device
+        _id -- ID of this device
+        vesync_dev -- the vesync device object to initialize from
+        """
+        EtekcityDevice.__init__(self, adapter, _id, vesync_dev)
+
+        self._type = ['OnOffSwitch', 'Light']
+        self.type = 'onOffLight'
+
+        if vesync_dev.dimmable_feature:
+            self.properties['brightness'] = EtekcityBulbProperty(
+                self,
+                'brightness',
+                {
+                    '@type': 'BrightnessProperty',
+                    'label': 'Brightness',
+                    'type': 'integer',
+                    'unit': 'percent',
+                    'minimum': 1,
```

Comment:
Not 0?

Reply:
Nope. According to the library, range is 1-100.